src/server/game/Grids/Notifiers/GridNotifiers.h

```diff
@@ -1321,10 +1322,18 @@ namespace Trinity
             }
 
         private:
-            WorldObject const& i_obj;
-            uint32 i_entry;
-            bool   i_alive;
-            float  i_range;
+            bool IsPersonalSpawnOfAnoter(Creature* creature)
+            {
+                if (TempSummon const* tempSummon = creature->ToTempSummon())
+                    if (tempSummon->IsVisibleBySummonerOnly() && i_obj.GetGUID() != tempSummon->GetSummonerGUID())
```

Comment:
GetSummonerGUID is not a good idea because there are tempoary summons that do not get a summonerguid or creatorguid assigned (wild summons for example) so you actually gotta check for  GetSummoner()->GetGUID

Reply:
This calls `TempSummon::GetSummonerGUID()`, it is fine